.circleci/config.yml

```
@@ -463,14 +463,16 @@ workflows:
       - just_simulate_sc_rehearsal_4:
           context:
             - circleci-repo-readonly-authenticated-github-token
+      - just_simulate_nested:
+          task: "eth/031-op-mainnet-pectra-defense"
+          context:
+            - circleci-repo-readonly-authenticated-github-token
 
-      # Simulation sequence for Superchain Version Sync (at the 27th of February 2025). The next task would be 31 but we need to wait until the task is 31 is merged.
-      # tasks: "030 031"
       - simulate_sequence:
           name: simulate_sequence_eth
           network: "eth"
-          tasks: "030"
-          block_number: "21938584" # If not specified, the latest block number is used.
+          tasks: "030 031"
```

030 was [executed already](https://github.com/ethereum-optimism/superchain-ops/blob/12437ff662bf63fab8e73105230d7c27903f69c3/tasks/eth/030-soneium-holocene-system-config-upgrade/README.md#L3) so this should be updated right?
```suggestion
          tasks: "031"
```

---

yeah, I think that's why it's failing.